N/NetCDF/NetCDF@julia-1.6/build_tarballs.jl

```diff
@@ -1,5 +1,6 @@
 include("../common.jl")
 
+ 
 # Offset to add to the version number.  Remember to always bumpt this.
 version_offset = v"0.2.0"
```

Comment:
The safest option may be to bump the offset number here, since we have it
```suggestion
version_offset = v"0.2.1"
```